c++/src/capnp/realtime.capnp

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2022 Cloudflare, Inc. and contributors
```

Comment:
I did not really know what to put there, I just copied from `stream.capnp`. Let me know if that should be something other than `Cloudflare` :sweat_smile: